package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --open=true",
```

Comment:
This allows for opening the page automatically when using `npm start` or `yarn start`